### ISSUE TITLE
Number requires a country to be specified.

### ISSUE THREAD

**Issue by DSpeichert** [ysfkaya/filament-phone-input#21]
Sometimes when a phone number isn't using a format recognized for a given country, there is an exception thrown instead of validation failure:

> Number requires a country to be specified.

Stacktrace:
> /app/vendor/propaganistas/laravel-phone/src/PhoneNumber.php:215
> /app/vendor/propaganistas/laravel-phone/src/PhoneNumber.php:111
> /app/vendor/propaganistas/laravel-phone/src/helpers.php:11
> /app/vendor/ysfkaya/filament-phone-input/src/Forms/PhoneInput.php:148
> /app/vendor/filament/support/src/Concerns/EvaluatesClosures.php:35

**Comment by ysfkaya** [comment]
Can you share the usage method in Filament and the usage method on the frontend? Also, how exactly did you save it to the database, what format did you use?

**Comment by DustinReagan** [comment]
I'm running into the same issue.

Here's a snippet:

```
Grid::make(4)->schema([
                            PhoneInput::make('main_phone')->label('Phone (Primary)')->columnSpan(2),
                            PhoneInput::make('mobile_phone')->label('Phone (Cell)')->columnSpan(2),

                        ])
 ```
 
 data:
 
 ```
 $data['main_phone']="212-975-4846"
 $data['mobile_phone']="732-735-5284"
 ```

**Comment by DustinReagan** [comment]
Not sure if this is helpful, but it appears the $country array is null in the `PhoneNumber` constructor, and this ends up causing `getCountry` to return null, which results in the exception being thrown.

<img width="880" alt="Screenshot 2023-10-11 at 7 43 42 PM" src="https://github.com/ysfkaya/filament-phone-input/assets/4032087/bca3ee4e-5f4e-4572-8f9f-2fcc6b093eec">

**Comment by ysfkaya** [comment]
I just released `2.1.0`. 

I updated the README and tried to explain why the error might occur. You can follow the instructions regarding the [Troubleshooting](https://github.com/ysfkaya/filament-phone-input?tab=readme-ov-file#troubleshooting) section.

Let me know if the problem persists

**Comment by ysfkaya** [comment]
I am closing this issue as there is no response. If the problem persists, you can proceed from the discussion section.
